crates/superposition_types/src/lib.rs

```diff
@@ -234,7 +246,7 @@ impl Display for RegexEnum {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         let regex = match self {
             Self::DefaultConfigKey => ALPHANUMERIC_WITH_DOT,
-            Self::DimensionName => ALPHANUMERIC_WITH_DOT,
+            Self::DimensionName => ALPHANUMERIC_WITHOUT_DOT,
```

Comment:
why are we changing this ?
we do intend to allow dimension names like `{key1}.{key2}`